BLE_BatteryLevel/source/main.cpp

```diff
@@ -43,7 +43,7 @@ class BatteryDemo : ble::Gap::EventHandler {
         _ble.init(this, &BatteryDemo::on_init_complete);
 
         _event_queue.call_every(500, this, &BatteryDemo::blink);
-        _event_queue.call_every(1000, this, &BatteryDemo::update_sensor_value);
+        _update_sensor_event = _event_queue.call_every(1000, this, &BatteryDemo::update_sensor_value);
```

Comment:
This line is not needed anymore since we do it in `onConnection()` now

Reply:
[Done](https://github.com/ARMmbed/mbed-os-example-ble/compare/3f15044595ebaf5a90644c562cc2904d2087c8f6..d1d8a73fcff47913ce0523927d6cba685d01018e).